### PR TITLE
Docs: Transfer in-memory-web-api into main repo

### LINKED ISSUE #40190: Docs: Transfer in-memory-web-api into main repo
# 📚 Transfer in-memory-web-api component documentation into main repo

### Description

In #37182 the [in-memory-web-api](https://github.com/angular/in-memory-web-api) module was moved into [this repo](https://github.com/angular/angular/tree/master/packages/misc/angular-in-memory-web-api). It seems the documentation has not yet been updated accordingly. There are two problems:

- [ ] At the moment the aio docs still have some links to the archived repo: `aio/content/guide/http.md` and `aio/content/tutorial/toh-pt6.md`
- [ ] The [readme](https://github.com/angular/in-memory-web-api/blob/master/README.md) is still only available in the old repo.

I propose to copy the [readme](https://github.com/angular/in-memory-web-api/blob/master/README.md) to `packages/misc/angular-in-memory-web-api/README.md` and update the links.

If you agree that this is the way forward let me know, I can create pr's.

### PR DESCRIPTION
**Pull request by SijmenHuizenga**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
In #37182 the in-memory-web-api module was moved into this repo. The documentation has not yet been updated accordingly. There are two problems:

* At the moment the aio docs still have some links to the archived repo: `aio/content/guide/http.md` and `aio/content/tutorial/toh-pt6.md`
* The readme is still only available in the old repo.

Issue Number: Fixes #40190

## What is the new behavior?

Make links in the docs point to the new location of the in-memory-web-api.

Copy the reamde into this repo with the following changes:
 * Removed Travis badges
 * Updated github links o piont to angular repo
 * Removed 'running tests' as it no longer relevant

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/misc/angular-in-memory-web-api/README.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+  InMemoryWebApiModule.forRoot(InMemHeroService, { delay: 1500 }), // 1.5 second delay
+```
+
+## Simple query strings
+Pass custom filters as a regex pattern via query string. 
+The query string defines which property and value to match.
+
+Format: `/app/heroes/?propertyName=regexPattern`
+
+The following example matches all names start with the letter 'j'  or 'J' in the heroes collection.
```
Comment (gkalpak) [inline packages/misc/angular-in-memory-web-api/README.md:206]:
```suggestion
The following example matches all names that start with the letter 'j' or 'J' in the heroes collection.
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @SijmenHuizenga! I've left a few minor comments. Otherwise lgtm 🚀

BTW, there are some typos in the commit messages:
- `links o piont to` --> `links to point to`
- `as it no longer` --> `as it is no longer`
- `Fixes: #40190` --> `Fixes #40190` (in both commits)

**Comment by gkalpak** [comment]
I forgot to mention, I think it is a good idea to include a note in the README that says this project used to live at https://github.com/angular/in-memory-web-api.

**Comment by SijmenHuizenga** [comment]
Thanks a lot for the corrections and suggestions @gkalpak! I've applied them all. Let me know if there is anything else you would like to see changed.

**Review by IgorMinar — APPROVED** [review]
LGTM. Thank you!

**Comment by IgorMinar** [comment]
@SijmenHuizenga  can you please rebase this PR and force push to your branch? thanks

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
@googlebot I consent.

**Pull request closed without merging**